.github/workflows/matlab.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
   ubuntu:
     name: AMD64 Ubuntu 20.04 MATLAB
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I think the explanation on the issue summary is great on why we have to use Ubuntu 20.04 here. Could we add a comment summarizing the issue here so we have a reminder on why are pinning this Ubuntu version?

Reply:
Added a comment explaining why we are pinning the operating system version to `ubuntu-20.04` explicitly.